aidants_connect_carto/apps/web/templates/layouts/nav.html

```diff
@@ -13,11 +13,11 @@ <h2>&nbsp;Carto</h2>
             Tous les lieux
           </a>
         </li>
-        <li class="nav__item">
+        <!-- <li class="nav__item">
```

Comment:
Pourquoi est-ce commenté ? :)

Reply:
J'ai caché les liens vers les fonctionnalités d'ajout & modif pour pas polluer l'interface de la v1, on garde ça sous le coude pour plus tard car ce n'est pas prioritaire. Et peut-être que toutes la partie edition pourrait se faire in fine sur une app séparée avec credentials ?

Reply:
Je comprends :)

Un problème que je trouve à cette approche est que ce contexte de compréhension n'est présent que dans ta tête, du coup l'éventuel·le dev du futur voyant cela se demanderait si c'était temporaire ? définitif ? abandonné ?  Si ça doit être maintenu ou si ça peut dégager sans crainte ? Ce genre de chose ^^

C'est en me plaçant dans cette perspective là qu'aujourd'hui je préfère laisser aussi peu de code commenté que possible, et en expliquant dans le commentaire _pourquoi_ il est là ^^